src/image/pixels.js

```diff
@@ -636,74 +657,86 @@ function parseFilterArgs(...args) {
 }
 
 /**
- * Get a region of pixels, or a single pixel, from the canvas.
- *
- * Returns an array of [R,G,B,A] values for any pixel or grabs a section of
- * an image. If no parameters are specified, the entire image is returned.
- * Use the x and y parameters to get the value of one pixel. Get a section of
- * the display window by specifying additional w and h parameters. When
- * getting an image, the x and y parameters define the coordinates for the
- * upper-left corner of the image, regardless of the current <a href="#/p5/imageMode">imageMode()</a>.
- *
- * Getting the color of a single pixel with get(x, y) is easy, but not as fast
- * as grabbing the data directly from <a href="#/p5/pixels">pixels[]</a>. The equivalent statement to
- * get(x, y) using <a href="#/p5/pixels">pixels[]</a> with pixel density d is
- * ```javascript
- * let x, y, d; // set these to the coordinates
- * let off = (y * width + x) * d * 4;
- * let components = [
- *   pixels[off],
- *   pixels[off + 1],
- *   pixels[off + 2],
- *   pixels[off + 3]
- * ];
- * print(components);
- * ```
- * See the reference for <a href="#/p5/pixels">pixels[]</a> for more information.
- *
- * If you want to extract an array of colors or a subimage from an p5.Image object,
- * take a look at <a href="#/p5.Image/get">p5.Image.get()</a>
+ * Gets a pixel or a region of pixels from the canvas. `get()` is easy to use
+ * but it's not as fast as <a href="#/p5/pixels">pixels</a>.
+ *
+ * The version of `get()` with no parameters returns the entire canvas.
```

Comment:
Probably worth noting that this version returns it as a `p5.Image` instead of an array, so it can be used with `image(...)` or `img.filter(...)`.